core/src/test/java/org/elasticsearch/search/suggest/completion/FuzzyOptionsTests.java

```diff
@@ -41,49 +47,54 @@ public static FuzzyOptions randomFuzzyOptions() {
-        builder.setFuzziness(original.getEditDistance())
-            .setFuzzyPrefixLength(original.getFuzzyPrefixLength())
-            .setFuzzyMinLength(original.getFuzzyMinLength())
-            .setMaxDeterminizedStates(original.getMaxDeterminizedStates())
-            .setTranspositions(original.isTranspositions())
-            .setUnicodeAware(original.isUnicodeAware());
+        builder.setFuzziness(original.getEditDistance()).setFuzzyPrefixLength(original.getFuzzyPrefixLength())
+                .setFuzzyMinLength(original.getFuzzyMinLength()).setMaxDeterminizedStates(original.getMaxDeterminizedStates())
+                .setTranspositions(original.isTranspositions()).setUnicodeAware(original.isUnicodeAware());
         switch (randomIntBetween(0, 5)) {
-            case 0:
-                builder.setFuzziness(randomValueOtherThan(original.getEditDistance(), () -> randomFrom(0, 1, 2)));
-                break;
-            case 1:
-                builder.setFuzzyPrefixLength(randomValueOtherThan(original.getFuzzyPrefixLength(), () ->
-                    randomIntBetween(1, 3)));
-                break;
-            case 2:
-                builder.setFuzzyMinLength(randomValueOtherThan(original.getFuzzyMinLength(), () ->
-                    randomIntBetween(1, 3)));
-                break;
-            case 3:
-                builder.setMaxDeterminizedStates(randomValueOtherThan(original.getMaxDeterminizedStates(), () ->
-                    randomIntBetween(1, 10)));
-                break;
-            case 4:
-                builder.setTranspositions(!original.isTranspositions());
-                break;
-            case 5:
-                builder.setUnicodeAware(!original.isUnicodeAware());
-                break;
+        case 0:
+            builder.setFuzziness(randomValueOtherThan(original.getEditDistance(), () -> randomFrom(0, 1, 2)));
+            break;
+        case 1:
+            builder.setFuzzyPrefixLength(randomValueOtherThan(original.getFuzzyPrefixLength(), () -> randomIntBetween(1, 3)));
+            break;
+        case 2:
+            builder.setFuzzyMinLength(randomValueOtherThan(original.getFuzzyMinLength(), () -> randomIntBetween(1, 3)));
+            break;
+        case 3:
+            builder.setMaxDeterminizedStates(randomValueOtherThan(original.getMaxDeterminizedStates(), () -> randomIntBetween(1, 10)));
+            break;
+        case 4:
+            builder.setTranspositions(!original.isTranspositions());
+            break;
+        case 5:
+            builder.setUnicodeAware(!original.isUnicodeAware());
+            break;
         }
         return builder.build();
     }
 
-    @Override
-    protected FuzzyOptions readFrom(StreamInput in) throws IOException {
-        return new FuzzyOptions(in);
+    /**
+     * Test serialization and deserialization
+     */
+    public void testSerialization() throws IOException {
+        for (int i = 0; i < NUMBER_OF_RUNS; i++) {
```

Comment:
I understand the concern, but CI runs are so infrequent now really and also if I run those tests from my IDE I usually really want to test more than one random permutation of the objects under test. This has caught so many things for me already in the past that I think its really worth it. Plus, this doesn't really add a lot of overhead, last time I looked at it these additional runs added almost nothing to the runtime of the test compared to initiallization. So to summarize: I think it buys a lot more to have more than one permutation per test than it costs.